docs/release/upgrading.md

```diff
@@ -1,5 +1,32 @@
 # Upgrading to DAOS Version 2.0
 
+
+## Upgrading DAOS from Version 2.0.x to Version 2.0.y
+
+Upgrading DAOS from one 2.0.x fix level to a newer 2.0.y fix level is
+supported as an offline update, maintaining the data in DAOS pools and
+containers.
+
+The recommended procedure for the upgrade is:
+
+- Ensure that there are no client applications with open pool connections.
+  If necessary, the `dmg pool evict` command can be used to disconnect
+  any active pool connections.
+- Stop the `daos_agent` daemons.
+- Stop the DAOS engines by running `dmg system stop`.
+- Stop the `daos_server` daemons.
+- Perform the RPM update to the new DAOS fix level.
+- Start the `daos_server` daemons.
+- Validate that all engines have started successfully,
+  for example using `dmg system query -v`.
+- Start the `daos_engine` daemons.
```

Comment:
Should it not be the 'daos_agent' daemons instead ?

Reply:
yes ... fixing